staging/src/k8s.io/apiserver/pkg/apis/config/v1alpha1/conversion.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/apiserver/pkg/apis/config"
+)
+
+// Important! The public back-and-forth conversion functions for the types in this generic
+// package with ComponentConfig types need to be manually exposed like this in order for
+// other packages that reference this package to be able to call these conversion functions
+// in an autogenerated manner.
```

Comment:
why is this? Is this a bug in conversion-gen? Or is it by design?

Reply:
The file creating Manually.
Like the comments, if no this file, `zz_conversion.go` in `pkg/apis` can't reference the `conversion function` in `k8s.io/<repo-name>`

Reply:
Am not convinced. This pretty much looks like a bug of conversion-gen.

Reply:
It probably is, but I hope we can merge this to unblock the ComponentConfig move process